html/Good_practices_in_Web_Dev/good-practices-in-web-dev.md

```diff
@@ -0,0 +1,248 @@
+# Good practices in Web-Dev
+
+To make sure quality user experience, your website must be displayed correctly across all the devices. To achieve this, we need to write scalable code and give the same output for all the devices.
+
+## what will it change?
+Following this will, of course, enhance the user experience once they visit your web platform. It will improve the readability of your code with the help of clear indentations, and depending upon the performance, it will also enhance the SEO rankings, pushing your platform to the top in search engines. It'll boost up the accessibility on the screen reader and other identical technologies.
```

Comment:
You're touching a lot of point here: Maybe update the theme of article around these points and do a little more research on how to achieve that.

Example:

Web Dev best practices for
- SEO
- Accessibility
- Readability
- Scalability (you can talk about modular code and other principle like DRY and SOLID)

Once the theme is set, you can expand around these and write under each of these headings.